src/probabilities_estimators/permutation_ordinal/spatial_permutation.jl

```diff
@@ -21,13 +23,34 @@ stencil = CartesianIndex.([(0,1), (1,1), (1,0)])
 est = SpatialSymbolicPermutation(stencil, x)
 ```
 
+or equivalently
+
+```julia
+data = [rand(50, 50) for _ in 1:50]
+x = data[1] # first "time slice" of a spatial system evolution
+stencil = [1 1; 1 1]
+est = SpatialSymbolicPermutation(stencil, x)
+```
+
 Here the stencil creates a 2x2 square extending to the bottom and right of the pixel
 (directions here correspond to the way Julia prints matrices by default).
 Notice that no offset (meaning the pixel itself) is always included automatically.
 The length of the stencil decides the order of the permutation entropy, and the ordering
 within the stencil dictates the order that pixels are compared with.
 The pixel without any offset is always first in the order.
 
+For rectangular/ cuboid stencils, one can also pass two `NTuple`s `extent` and `lag`,
+from which an appropriate stencil will be created. 
+`extent` defines how many points should be considered in each direction, and `lag`
```

Comment:
maybe say "`extent[i]` defines how many points should be considered along the `i`-th axis, and `lag[i]` ..."